ai/anthropic/v0/main.go

```diff
@@ -0,0 +1,176 @@
+package anthropic
+
+import (
+	"context"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	"github.com/instill-ai/component/base"
+)
+
+const (
+	textGenerationTask = "TASK_TEXT_GENERATION_CHAT"
+	cfgAPIKey          = "api_key"
+	host               = "https://api.anthropic.com"
+	messagesPath       = "/v1/messages"
+)
+
+var (
+	//go:embed config/definition.json
+	definitionJSON []byte
+	//go:embed config/setup.json
+	setupJSON []byte
+	//go:embed config/tasks.json
+	tasksJSON []byte
+
+	once sync.Once
+	comp *component
+)
+
+type component struct {
+	base.Component
+}
+
+// Init returns an implementation of IComponent that implements the greeting
+// task.
```

Comment:
Important
Let's remove all unnecessary comments!
Please take the comments out if you think it does not explain further information.

Same as other comments

Reply:
fixed